app/views/profiles/_profile.html.erb

```diff
@@ -1,15 +1,15 @@
 <% if profile %>
-  <% tooltip_empty = !profile.iso_languages.present? && profile.city.nil? && profile.willing_to_travel.nil? && profile.nonprofit.nil? %>
+  <% tooltip_empty = !profile[:iso_languages].present? && profile[:city].nil? && profile[:willing_to_travel].nil? && profile[:nonprofit].nil? %>
   <% title = render partial: "profiles/profile_tooltip", locals: { profile: profile } %>
   <div class="col-sm-6 col-md-6 col-lg-3 my-3 profile-box">
-    <%= profile_image_link(profile) %>
+    <%= profile_image_link(Profile.find(profile[:id])) %>
```

Comment:
Why not using profile? Here you have to grep it again from the DB

Reply:
In this case `profile` is not the object itself but the hash with the limited attributes. But the `profile_image_link` method needs the profile instance, doesn't it?